twilight-model/src/guild/mod.rs

```diff
@@ -103,6 +103,8 @@ pub struct Guild {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub max_presences: Option<u64>,
     #[serde(skip_serializing_if = "Option::is_none")]
+    pub max_stage_video_channel_users: Option<u64>,
```

Comment:
Can we add documentation here? I think copying what was on the cache method is fine 👍